pages-config.json

```diff
@@ -14,6 +14,7 @@
   "assetRoot":        "/guides-template",
   "fileLockWaitTime": 30000,
   "fileLockPollTime": 1000,
+  "secretKeyFile":    "/home/ubuntu/.18f-pages.secret",
```

Comment:
Not a blocker for this PR, but it's probably worth distinguishing the 18F-specific configuration from the general files/components, e.g. by moving it to a different repository, or at least making a note of it in the README.